src/views/students/Students.jsx

```diff
@@ -1,25 +1,37 @@
-import React from 'react';
-import Container from 'react-bootstrap/Container';
-import Table from 'react-bootstrap/Table';
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import Container from "react-bootstrap/Container";
+import Table from "react-bootstrap/Table";
 
 /**
  * Implement the read students information.
- * 
+ *
  * The model definition is:
  * [{
  *  firstName: string,
  *  lastName: string,
  *  email: string,
  *  program: string
  * }]
- * 
+ *
  * Replace the hardcoded data with the data from the following endpoint:
  * GET https://api-demo.kingslandtesting.com/student/
  */
+
 const Students = () => {
+  const [students, setStudents] = useState([]);
+
+  useEffect(() => {
+    axios
```

Comment:
Good proper use of `.then()`.

In comparison with your previous promise implementation, observe that your function here does not need `await` anymore and your code works just fine.